README.md

```diff
@@ -11,111 +16,148 @@
+The safest way to install Baritone is to install a release .jar from this repository 
+and place it in your .minecraft\mods\ folder.
 
-The easiest way to install Baritone is to install [Impact](https://impactclient.net/), which comes with Baritone. The second easiest way (for 1.12.2 only) is to install the v1.2.* `api-forge` jar from [releases](https://github.com/cabaletta/baritone/releases). **For 1.12.2 Forge, just click [here](https://github.com/cabaletta/baritone/releases/download/v1.2.15/baritone-api-forge-1.2.15.jar)**. Otherwise, see [Installation & setup](SETUP.md). Once Baritone is installed, look [here](USAGE.md) for instructions on how to use it.
+The most recent version of Minecraft that Baritone currently supports is *1.16.5* ,
+Baritone will be updated to 1.17.~ soon.
 
-For 1.15.2, [click here](https://www.youtube.com/watch?v=j1qKtCZFURM) and see description. If you need Forge 1.15.2, look [here](https://github.com/cabaletta/baritone/releases/tag/v1.5.3), follow the instructions, and get the `api-forge` jar.
+## Getting Started
 
-For 1.16.5, [click here](https://www.youtube.com/watch?v=_4eVJ9Qz2J8) and see description. If you need Forge or Fabric 1.16.5, look [here](https://github.com/cabaletta/baritone/releases/tag/v1.6.3) and get the `api-forge` or `api-fabric` jar. **For 1.16.5 Fabric, just click [here](https://github.com/cabaletta/baritone/releases/download/v1.6.3/baritone-api-fabric-1.6.3.jar)**.
+- #### [Features](FEATURES.md)
 
-This project is an updated version of [MineBot](https://github.com/leijurv/MineBot/),
-the original version of the bot for Minecraft 1.8.9, rebuilt for 1.12.2 through 1.16.5. Baritone focuses on reliability and particularly performance (it's over [30x faster](https://github.com/cabaletta/baritone/pull/180#issuecomment-423822928) than MineBot at calculating paths).
+- #### [Installation & setup](SETUP.md)
 
-Have committed at least once a day from Aug 1, 2018, to Aug 1, 2019.
+- #### [API Javadocs](https://baritone.leijurv.com/)
 
-1Leijurv3DWTrGAfmmiTphjhXLvQiHg7K2
+- #### [Settings](https://baritone.leijurv.com/baritone/api/Settings.html#field.detail)
 
-# Getting Started
+- #### [Usage (chat control)](USAGE.md)
 
-Here are some links to help to get started:
+## Baritone API
 
-- [Features](FEATURES.md)
+There is extensive documentation for the Baritone API,
+[JavaDocs](https://baritone.leijurv.com/).
+Please note that usage of anything located outside the ``baritone.api`` package is unsupported by the API release jar.
 
-- [Installation & setup](SETUP.md)
+Below is an example of changing settings, and then pathing to an X/Z goal.
 
-- [API Javadocs](https://baritone.leijurv.com/)
+```
+BaritoneAPI.getSettings().allowSprint.value = true;
+BaritoneAPI.getSettings().primaryTimeoutMS.value = 2000L;
 
-- [Settings](https://baritone.leijurv.com/baritone/api/Settings.html#field.detail)
+BaritoneAPI.getProvider().getPrimaryBaritone().getCustomGoalProcess().setGoalAndPath(new GoalXZ(10000, 20000));
+```
```

Comment:
Really this sort of thing should be in a separate file I think, not many of the people who will read this readme will be interested in how the baritoneAPI works.

Reply:
Agree on that, if there is more than just this little sample in the new file.